src/releng_docs/deploy/regular.rst

```diff
@@ -65,16 +109,20 @@ Protocal that we follow is:
 #. Push to ``production``. Create a merge commit of master branch and tag it.
    Don't forget to push just created tag.
 
+  TODO: is this the correct branching model
+
    .. code-block:: console
 
        $ git clone git@github.com/mozilla/release-services.git
```

Comment:
Nit: Same as above, maybe remove the `$` prefixes for easier block copy/paste/execute.